python/src/equistore/labels.py

```diff
@@ -251,6 +251,41 @@ def asarray(self):
     def __contains__(self, label):
         return self.position(label) is not None
 
+    @staticmethod
+    def arange(*args, name="name"):
+        """
+        Creates a very simple `Labels` object.
+
+        The resulting `Labels` only contain one label, whose name
+        is `name`, unless otherwise specified. The values of the
+        labels depend on a set of integer arguments, whose
+        meaning is the same as those of the `numpy.arange` function.
+        As a result, this function only accepts one to three such
+        integer parameters.
+        """
+
+        args_len = len(args)
+        if args_len == 0:
+            raise ValueError(
+                "please provide at least one integer for `Labels.arange()`"
+            )
+        if args_len > 3:
+            raise ValueError(
+                f"the maximum number of integer arguments accepted by \
+                `Labels.arange()` is 3. {args_len} were provided"
+            )
+        for arg in args:
+            if not np.issubdtype(type(arg), np.integer):
+                raise ValueError(
+                    "all numbers provided to `Labels.arange()` must be integers"
+                )
```

Comment:
I think you don't need any checks here. Just pass everything to `np.arange`. They will handle it no?

Reply:
I feel like our own error messages can be more informative

Reply:
fine for me